tasks/pdr-cleanup/pdr_cleanup.py

```diff
@@ -0,0 +1,97 @@
+"""Task for archiving PDRs."""
+
+import json
+import logging
+import os
+from datetime import datetime
+
+import boto3
+from cumulus_logger import CumulusLogger
+from run_cumulus_task import run_cumulus_task
+
+logger = CumulusLogger("pdr-cleanup", logging.INFO)
+
+
+def cleanup_pdr(event: dict, context: dict):
+    """Task to archive PDRs.
+
+    Args:
+        event (dict): A lambda event object
+        context (dict): An AWS Lambda context
+
+    """
+
+    logger.info("## EVENT OBJ \n" + json.dumps(event))
+
+    provider = event["config"]["provider"]
+    pdr = event["input"]["pdr"]
+
+    if len(event["input"]["failed"]) == 0:
+        move_pdr(provider, pdr)
+    else:
+        logger.info("PDR failed to ingest all granules successfully, NOT archiving")
+        raise Exception(
+            "PDR failed to ingest all granules successfully\n"
+            f"Ingest Granule workflow failure count: {len(event['input']['failed'])}"
+        )
+    return event["input"]
```

Comment:
Is the output the input here or do we also need to provide where the PDR was moved to in the output message? Are there any other tasks that would run after this one?